app/assets/javascripts/organizations.js

```diff
@@ -1,11 +1,17 @@
-$(document).pjax('a.organization-link', '#organization-content', {push: false});
-$(document).pjax('a.edit-link-pjax', '#organization-content', {push: false});
-$(document).pjax('a.update-link-pjax', '#organization-content', {push: false});
+$(document).pjax('a.organization-link', '#organization-content', {push: true});
+$(document).pjax('a.edit-link-pjax', '#organization-content', {push: true});
+$(document).pjax('a.update-link-pjax', '#organization-content', {push: true});
 
 $(document).on('submit', '#edit-form form', function(event) {
   $.pjax.submit(event, '#organization-content', {push: false});
 });
 
+$(document).ready(function() {
+  if (('.org-address').length >= 1 ) {
+    displayMap($('.org-address').text());
```

Comment:
We can ignore this for now, but in the future we should make sure we're requiring things correctly so we don't run into any dependency issues.